src/main/java/com/gorlah/kappabot/integration/discord/DiscordBot.java

```diff
@@ -17,10 +15,10 @@
 
 import javax.annotation.PostConstruct;
 import javax.security.auth.login.LoginException;
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Map;
 
+@Slf4j
 @Component
 @RequiredArgsConstructor
 public class DiscordBot extends ListenerAdapter {
```

Comment:
This class looks a lot better now.

Reply:
Thanks! It looks even better with the changes I have in my local. :wink: